src/ray/raylet/task_dependency_manager.cc

```diff
@@ -242,11 +243,24 @@ void TaskDependencyManager::TaskPending(const Task &task) {
       }
     }
 
-    // Acquire the lease for the task's execution in the global lease table.
-    AcquireTaskLease(task_id);
+    // For non-actor tasks, acquire the lease for the task's execution in the
+    // global lease table.
+    if (!task.GetTaskSpecification().IsActorTask()) {
+      AcquireTaskLease(task_id);
+    }
   }
 }
 
+void TaskDependencyManager::AddTaskLeaseData(const TaskID &task_id, int64_t lease_period,
+                                             uint64_t actor_version) {
```

Comment:
Isn't the actor version -1 in some places? So why `uint64_t`? Same comment in one or two other places.